### PR TITLE
Jir 100

### PR DESCRIPTION
**Pull request by Shuja-Siddiqui**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -631,6 +633,14 @@ function App() {
...
                                   element={<DeleteUser />}
                                 />
+                                <Route
+                                  path="/umeedwar-nazim"
+                                  element={<UmeedWarNazim />}
+                                />
+                                {/* <Route
+                                  path="/temp"
+                                  element={<Temp />}
+                                /> */}
```
Comment (i-abdul-raheem) [inline src/App.jsx:643]: Remove this temp route

File: src/Temp.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+//       </button>
+//     </>
+//   );
+// };
+
+
+
+// const a = {}
+// a['b'] = {}
+// a['b']['c'] = 123;
```
Comment (i-abdul-raheem) [inline src/Temp.jsx:101]: Remove this file

File: src/api/instrance.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ const instance = axios.create({
   baseURL:
     env === "production"
       ? "http://192.168.0.117:5000/api/v1"
-      : "http://192.168.0.113:5000/api/v1",
+      : "http://localhost:5000/api/v1",
```
Comment (i-abdul-raheem) [inline src/api/instrance.js:7]: Update this to production base url

**Review by i-abdul-raheem — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (JsCodexx) [reply src/App.jsx:643]: removed

File: src/api/instrance.js (hunk 3 above)
Comment (JsCodexx) [reply src/api/instrance.js:7]: updated

**Review by i-abdul-raheem — APPROVED** [review]

**Pull request merged**